docs/en/source/systemOverview.rst

```diff
@@ -0,0 +1,39 @@
+*****************************************
+Architecture
```

Comment:
again: NGB / IoT Broker -> Scorpio Broker
"will be implemented" --> have been implemented as
as above, I would avoid using the term "IoT Entities" as this may be misleading with respect to Entities in NGSI-LD - in general we have applications that can have the role of producers, consumers or both
Storage Connectors: make clear what is currently the case and describe alternative future options
Consistently capitalize HTTP